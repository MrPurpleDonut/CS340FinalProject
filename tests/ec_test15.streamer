switch (args[0])
{
  case "Monday":
    print "A bit tired; the week is just beginning.";
  case "Thursday":
    print "Almost done with the week!";
  case "Friday":
    print "The weekend is here!";
  case "Saturday":
    print "Still enjoying the weekend!";
  case "Sunday":
    print "We start all over tomorrow...";
  default:
    print "Just an ordinary day.";
}

