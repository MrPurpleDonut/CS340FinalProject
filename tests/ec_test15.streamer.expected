Program parsed correctly!
---------------------------
--------Printing AST-------
---------------------------
Program
    Function List
    Statement List
        Switch Statement
            List Access
                Identifier: args
                Number: 0.0
            Case List
                Case
                    Quoted string: "Monday"
                    Print Statement
                        Quoted string: "A bit tired; the week is just beginning."
                Case
                    Quoted string: "Thursday"
                    Print Statement
                        Quoted string: "Almost done with the week!"
                Case
                    Quoted string: "Friday"
                    Print Statement
                        Quoted string: "The weekend is here!"
                Case
                    Quoted string: "Saturday"
                    Print Statement
                        Quoted string: "Still enjoying the weekend!"
                Case
                    Quoted string: "Sunday"
                    Print Statement
                        Quoted string: "We start all over tomorrow..."
                Default Case
                    Print Statement
                        Quoted string: "Just an ordinary day."
